### ISSUE TITLE
Installation: An error occurred while trying to replace the existing file: kprocesshacker.sys

### ISSUE THREAD

**Issue by WiliTest** [processhacker/processhacker#449]
> C:\Program Files\Process Hacker 2\kprocesshacker.sys
> An error occurred while trying to replace the existing file:
> DeleteFile failed; code 5.
> Access is denied.
> Click Retry to try again, Ignore to skip this file (not recommended), or Abort to cancel installation.

I can't delete the file manually and using an admin PowerShell  `Remove-Item -Path "C:\Program Files\Process Hacker 2\kprocesshacker.sys" -Force`

**Comment by Biswa96** [comment]
Stop the kernel mode driver with `sc stop kprocesshacker3`.

**Comment by WiliTest** [comment]
Thank you! Before I saw your answer, I solved it by killing SearchUI.exe (open handle found using ressource monitor tool). 

Unrelated question: how can I display the search field (I updated after reseting the settings, I don't find it anymore on the top right)

**Comment by WiliTest** [comment]
thanks to reopen it, actually it did not work. 
Process hacker was not installed succesfully, I saw it when I looked for the plugins. Then I tried to uninstall it but it was not installed.

I tried to delete the files of `C:\Program Files\Process Hacker 2` but failed to delete kprocesshacker.sys (same error than above)

These PowerShell did not work:
sc stop kprocesshacker
sc stop kprocesshacker2
sc stop kprocesshacker3
